discord/interactions.py

```diff
@@ -1380,7 +1420,7 @@ async def _locked_response(self, coro: Coroutine[Any, Any, Any]) -> None:
             if self.is_done():
                 coro.close()  # cleanup un-awaited coroutine
                 raise InteractionResponded(self._parent)
-            await coro
+            return await coro
```

Comment:
Missing Returns section in docstring (even if private, still), as well as .. versionchanged